packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2887,6 +3007,45 @@ class IgnorePointer extends SingleChildRenderObjectWidget {
   }
 }
 
+/// Holds opaque meta data in the render tree.
+class MetaData extends SingleChildRenderObjectWidget {
+  /// Creates a widget that hold opaque metadata.
```

Comment:
I went with "meta data" everywhere because that's what the code went with, but metadata seems to be accepted usage as well.